apps/web/composables/useItemTable/useItemTable.ts

```diff
@@ -28,17 +45,107 @@ export const useItemsTable: UseItemTableReturn = () => {
     }
 
     const items: StorageObject[] = response.data ?? [];
-
     state.value.data = items;
     cachedImages.value = items;
   };
-
   const headers = ref([
     { title: 'File name', key: 'key' },
     { title: 'Image size', key: 'size' },
     { title: 'Last change', key: 'lastModified' },
   ]);
 
+  const makeTempKey = (name: string) => `__uploading__:${Date.now()}:${name}`;
+
+  const addPlaceholderFirst = (key: string, size: number) => {
+    const ph: StorageObject = {
+      key,
+      eTag: '',
+      size: String(size),
+      lastModified: new Date().toISOString(),
+      storageClass: UPLOADING_CLASS,
+      publicUrl: '',
+      previewUrl: '',
+    };
```

Comment:
```suggestion
    const placeholder: StorageObject = {
      key,
      eTag: '',
      size: String(size),
      lastModified: new Date().toISOString(),
      storageClass: UPLOADING_CLASS,
      publicUrl: '',
      previewUrl: '',
    };
    state.value.data = [placeholder, ...state.value.data];
    cachedImages.value = [placeholder, ...cachedImages.value];
```
Abbreviating image to img is already somewhat borderline in my view, but let's not overdo it. I shouldn't have to read the method name to know what the constant stands for.